nixos/modules/services/networking/networkmanager.nix

```
@@ -523,6 +512,9 @@ in
       [ "networking" "networkmanager" "fccUnlockScripts" ]
       [ "networking" "modemmanager" "fccUnlockScripts" ]
     )
+    (mkRemovedOptionModule [ "network" "networkmanager" "enableStrongSwan" ]
+      "This option was removed. Pass `pkgs.networkmanager_strongswan` into `networking.networkmanager.plugins` instead."
```

```suggestion
      "Pass `pkgs.networkmanager_strongswan` into `networking.networkmanager.plugins` instead."
```
Otherwise that part would duplicated https://github.com/NixOS/nixpkgs/blob/dcc4da22e1bf5ed04220e7c5ddaa80abc0dee646/lib/modules.nix#L1497